addons/website/static/src/js/editor_helper.js

```diff
@@ -0,0 +1,11 @@
+/** @odoo-module */
+
+import { loadWysiwyg } from 'web_editor.loader';
+import websiteRootInstance from '../js/content/website_root_instance';
+
+// FIXME: Should be made more robust to ensure we're in edit mode.
+if (window.parent !== window) {
+    // FIXME: Probably don't need the entire package, most likely just styles.
+    loadWysiwyg(['website.compiled_assets_wysiwyg']);
+    window.websiteRootInstance = websiteRootInstance;
```

Comment:
Sorry, I don't remember why it is needed to load the wysiwyg assets from the iframe, is it still necessary ? 
I understand that we want the want the website root instance in the window, but can we do that in the redirect.js script ?

Reply:
Ah I should add a comment: For now it's needed because I haven't sorted which functions from the editor are actually needed inside the iframe. Some styles and logic (like the dropzones for exemple) need to be inside the iframe. Eventually, I would like to remove it or trim it down so that the iframe content can potentially load quicker.

Reply:
Ok, thanks ! I guess we can take a closer look at that later :+1: